index.bs

```diff
@@ -252,21 +252,22 @@ A <dfn>lock request</dfn> represents a pending request for a [=lock-concept|lock
 
 A [=lock request=] is a tuple of (<dfn>agent</dfn>, <dfn>clientId</dfn>, <dfn>origin</dfn>, <dfn>name</dfn>, <dfn>mode</dfn>, <dfn>promise</dfn>).
 
-Each origin has an associated <dfn>lock request queue</dfn>, which is a [=queue=] of [=lock requests=].
+Each origin has an associated <dfn>lock request queue set</dfn>, which is a [=set=] of [=lock request queues=] (a [=lock request queue=] being a [=queue=] of [=lock requests=]) where each queue is associated with a particular resource name.
```

Comment:
This sounds like a [map](https://infra.spec.whatwg.org/#maps) ? That would formalize the key/value nature (key being the name, value being the lock request queue)?

This needs a `<dfn>` for _lock request queue_, and that should be moved to its own line. Defining it in () makes the sentence hard to parse.